### PR TITLE
Readme for composable-traits

### PR DESCRIPTION
**Pull request by flipchan**
Basic read me for composable-traits

### CODE REVIEW

**Comment by ComposableFiB0t** [comment]
Task linked: [CU-1x0tqzc Readme for Composable-traits](https://app.clickup.com/t/1x0tqzc)

File: frame/composable-traits/readme.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Composable Traits   
+
+#### Composable-traits allow us to write simple interfaces to thirdparty pallet functions. 
+
+
+In order to have a clean source code and avoid adding unnecessary frame Config traits additions(example: ```rust pub trait Config: frame_system::Config + thirdpartycrate::Config {  ```), composable-traits comes to the rescue.
```
Comment (Br1ght0ne) [inline frame/composable-traits/readme.md:6]: Either use single backticks without lang name or make a multiline block

```suggestion
In order to have a clean source code and avoid adding unnecessary frame Config traits additions(example: ```pub trait Config: frame_system::Config + thirdpartycrate::Config {  ```), composable-traits comes to the rescue.  
```

File: frame/composable-traits/readme.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+#### Composable-traits allow us to write simple interfaces to thirdparty pallet functions. 
+
+
+In order to have a clean source code and avoid adding unnecessary frame Config traits additions(example: ```rust pub trait Config: frame_system::Config + thirdpartycrate::Config {  ```), composable-traits comes to the rescue.  
+Using traits we can easily import and use functions and logic from thirdparty frames without importing and configuring the thirdparty frame 
+functions in lib.rs.
+
+The logic for this is pretty simple, all we need to do is to create a function in our trait file that returns the same output as the original function. 
+After we have created a function inside our custom trait that returns the same output, we can simply import it from composable_traits( ```rust use composable_traits::newtrait::Ourtrait ``` ) and call it with a ```rust T::Ourtrait::customfunction() ```.
```
Comment (Br1ght0ne) [inline frame/composable-traits/readme.md:11]: Same as above

**Review by Br1ght0ne — COMMENTED** [review]
LGTM, see comments.

**Comment by KaiserKarel** [comment]
@hussein-aitlahcen can we rebase or close this?

**Pull request closed without merging**